### ISSUE TITLE
Bug Spider Web and Egg

### ISSUE THREAD

**Issue by Pedroguarana** [otsolutions/Vanaheim#19]
A Spider Web e o Spider Egg não estão quebrando com a arma.

**Comment by RafaelPereiraa** [comment]
Os spider eggs são quebrados dando 'use' neles, na verdade.

Quanto a Spider Web, vou dar uma olhada mais tarde. Chegou a testar com o firebug também?

**Comment by RafaelPereiraa** [comment]
Já resolvi aqui... agora só preciso saber como faço pra mandar pra cá sem comprometer o resto... visto que trabalho na mesma pasta que jogo/testo coisas no server...

**Comment by Pedroguarana** [comment]
opa, não cheguei a testar o firebug. Eu coloquei pra dar use na teia tbm pra contornar o problema. mas se você conseguir liberar o que você fez vai ser de grande ajuda.

**Comment by RafaelPereiraa** [comment]
Cara, então, se tu for no meu fork nesse commit: [https://github.com/RafaelPereiraa/Vanaheim/commit/0fc0de8540e26a6fb71de0e0434cc29dbabde923](url) tem as alterações necessárias... Na verdade, falando agora, me caiu a ficha que não arrumei a questão do firebug, e acabei gerando alguns outros probleminhas quanto ao "destruir" itens usando a arma... mas sobre os spider webs e spider eggs tá funcionando, já com os storages pra tibia tales e tudo mais.

É relativamente simples de resolver.

**Comment by RafaelPereiraa** [comment]
Cara, então, se tu for no meu fork nesse commit: [Spider Web/Eggs](https://github.com/RafaelPereiraa/Vanaheim/commit/0fc0de8540e26a6fb71de0e0434cc29dbabde923) tem as alterações necessárias... Na verdade, falando agora, me caiu a ficha que não arrumei a questão do firebug, e acabei gerando alguns outros probleminhas quanto ao "destruir" itens usando a arma... mas sobre os spider webs e spider eggs tá funcionando, já com os storages pra tibia tales e tudo mais.

É relativamente simples de resolver.

**Comment by brunominervino** [comment]
@RafaelPereiraa é necessário abrir um pull request para que eu avalie e aprove. Da uma olhada nesta documentação do GitHub: https://help.github.com/articles/using-pull-requests/

**Comment by RafaelPereiraa** [comment]
Sim, sim, mas tipo... eu mexi em outras coisas sabe? Que não fazem sentido/mudam em nada, são apenas ajustes pra minha jogabilidade mesmo. Tipo diminuição de exhaust etc.

Tu pode escolher o que vai ser adicionado na hora de aceitar?

**Comment by brunominervino** [comment]
Não, eu não posso editar o seu pull request
